### ISSUE TITLE
Hint/explain that the own system address book contact can be edited in settings

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/contacts#3405]
### Is your feature request related to a problem? Please describe.

As a user I might discover the *system address book* by chance, see my own information in there and wonder why I can't change anything.

### Describe the solution you'd like

Detect the system address book and that the displayed card is *me*. Then show a hint or even an *Edit* action, but instead of editing the contact (it's read-only) take me to my profile settings.

### Additional context

https://github.com/nextcloud/server/issues/19575

**Comment by jancborchardt** [comment]
@ChristophWurst yeah, really good idea! The button can just be where the "Edit" button normally is, but be labeled "Edit profile".
